sway-lsp/src/server.rs

```diff
@@ -375,31 +375,39 @@ mod tests {
     use tower::{Service, ServiceExt};
 
     use super::*;
-    use futures::stream::StreamExt;
-    use tower_lsp::jsonrpc::{self, Request, Response};
-    use tower_lsp::LspService;
+    use serial_test::serial;
+    use tower_lsp::{
+        jsonrpc::{self, Id, Request, Response},
+        ExitedError, LspService,
+    };
+
+    fn sway_workspace_dir() -> PathBuf {
+        env::current_dir().unwrap().parent().unwrap().to_path_buf()
+    }
+
+    fn e2e_language_dir() -> PathBuf {
+        PathBuf::from("test/src/e2e_vm_tests/test_programs/should_pass/language")
+    }
 
     #[allow(dead_code)]
```

Comment:
Why is this needed?

Reply:
just a convenience function as this path is called in 2 other places.